xlson/__init__.py

```diff
@@ -63,8 +63,23 @@ def __init__(self, **kwargs: Dict) -> None:
             bind_dict: Dict[str, Any] = kwargs.get("bind", {})
 
             for key, value in bind_dict.items():
+                # Handle bind::constraint field
+                if key == "constraint":
+                    if value.startswith("regex"):
+                        val = value.split("'")
+                        regex_val = val[1]
+                        elements["v_regex"] = {
+                            "value": regex_val,
+                            "err": bind_dict.get("jr:constraintMsg"),
+                        }
+                    else:
+                        elements["v_regex"] = {
```

Comment:
Only a regex constraint should get a `v_regex` key. We have different constraints which we should treat differently. Let's only handle regex in this PR.